### ISSUE TITLE
reverse engineering is slept on

### ISSUE THREAD

**Issue by deltanedas** [DeltaV-Station/Delta-v#1882]
## Description
in most rounds the RE machine is never even turned on

there are only a handful of things anyone actually bothers reverse engineering:
- non researchable items that are good for mass producing: the valid suits, handheld crew monitor (an objective farm lol) and RCD ammo (saves logi money)
- tier 3s you get somehow like the QSI on the clown shuttle (rip) or if you wipe your server then go down a different discipline

you *can* reverse engineer things like drills but they are so easy to both research and find more of that theres no reason to waste half an hour reverse engineering it.

i think reverse engineering something should unlock the recipe for its upgrade, some ideas:
- a 2-charge sci flash unlocks blueprint for a proper flash
- lantern unlocks flash lantern, would have to be very expensive though
- freezer unlocks hellfire freezer
- drill unlocks diamond drill
- regular jetpack unlocks void jetpack
- magboots or moonboots unlock speedboots at a very high difficulty
- antique/hyper cells from expeditions unlock microreactor cell
- ore bag unlocks bluespace ore bag, same for plant bag
- tier 3s being reproducable via RE should stay as-is for the possibility of wiping the server

having some t3s unlocked by REing stuff would also loosen t3 industrial's iron grip on research since you aren't automatically a goober for picking something else

basically it would be a parallel for research, probably to focus on things you don't need as urgently:
- a salv gives epi an ore bag and a drill to RE incase they have bad luck with artifacts
- botany gives epi a plant bag to RE since civ is rarely rushed

another thing to consider is that *you can make multiple RE machines*, so if RE was made to be more useful people might make multiple of them to RE multiple things at once

funny side note, you can RE the RE board, even though to do that you literally have to have it researched to make a second board, theres no other way

**Comment by Avalon-Proto** [comment]
Honestly I'd love if more things were RE-able as well. Say. RE Cap's laser to make Advanced laser pistols, so on so forth

**Comment by deltanedas** [comment]
the bad part of REing caps laser and other sensitive things is that 100% the RE machine is just getting moved to mystas room, the vault or caps room because nobody trusts epi with it, this already happens with suits

**Comment by Lyndomen** [comment]
I like the idea to make it a third avenue for research. How do we make it more interactive and engaging for people? Rn it's kinda waiting ops

**Comment by deltanedas** [comment]
maybe adding a third analysis mode that:
- wont destroy the item
- fast like with safety off
- requires random tools/parts/materials to be given to it after every step so its more engaging

if you want to just fire and forget turn off safety, if you want it to not get destroyed do the minigame mode

**Comment by therealDLondon** [comment]
New oracle just dropped:
"I DEMAND ONE WRENCH FOR STAGE 2 ANALYSIS!"
